debian/clickhouse-client.postinst

```diff
@@ -4,6 +4,3 @@ set -e
 CLICKHOUSE_USER=${CLICKHOUSE_USER=clickhouse}
 
 mkdir -p /etc/clickhouse-client/conf.d
-
-# user created by clickhouse-server package
-chown -R ${CLICKHOUSE_USER} /etc/clickhouse-client || true
```

Comment:
Maybe remove old preprocessed configs to avoid confusion (need to do it extra carefully)?